test_pytest_rerunfailures.py

```diff
@@ -511,3 +511,35 @@ def test_only_rerun_flag(testdir, only_rerun_texts, should_rerun):
     assert_outcomes(
         result, passed=num_passed, failed=num_failed, rerun=num_reruns_actual
     )
+
+
+@pytest.mark.parametrize(
+    "condition, expected_reruns",
+    [
+        (1 == 1, 2),
+        (1 == 2, 0),
+        (True, 2),
+        (False, 0),
+        (1, 2),
+        (0, 0),
+        ("'non-empty'", 2),
+        ("''", 0),
```

Comment:
it seems both f-strings and `format()` stripped the double quotes, so the single quotes was my solution.

Reply:
:+1: